getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d808a92c-d36d-4291-a4ad-b12bceb62fec",
+   "metadata": {},
+   "source": [
+    "# Getting Started Azure OpenAI with LangChain in Python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6689d50a-d463-4ad1-8968-180f93bd2f8e",
+   "metadata": {},
+   "source": [
+    "The purpose of this notebook is to provide a step-by-step guide on how to set up with Azure OpenAI and use the available models through the LangChain framework in Python. Additionally, it will demonstrate, through examples, how to implement models and utilize available features."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b8202de6-7d14-44e6-b378-879d26f95ce3",
+   "metadata": {},
+   "source": [
+    "## Obtaining Keys and Endpoints"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b21c348-6a01-41d4-9df5-0d1d633367ef",
+   "metadata": {},
+   "source": [
+    "Using the [**Azure sign up page**](https://azure.microsoft.com/en-us/pricing/purchase-options/azure-account) one can sign up and create an Azure OpenAI resource, giving access to all necessary credentials."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff61b81a-5c29-4da1-818f-fe3c2b1f9280",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "#### Installing and Importing Dependencies\n",
+    "Microsoft [**recommends**](https://github.com/microsoft/vscode-jupyter/wiki/Installing-Python-packages-in-Jupyter-Notebooks) using:\n",
+    "\n",
+    "-  %pip for installing within Jupyter or IDE environments.\n",
+    "-  %conda for installing within Conda environments.\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "448ce3be-b98f-4d46-8815-36fda94e9d30",
+   "metadata": {},
+   "source": [
+    "#### Install the most current version of `langchain` and `langchain_openai`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "23716db9-d8c8-4682-b662-0de8c185bd07",
+   "metadata": {},
+   "source": [
+    "`%pip install -U langchain langchain_openai`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f634643-841b-47fb-ba55-ff058e863528",
+   "metadata": {},
+   "source": [
+    "#### Install Packages in a Virtual Environment (Optional)\n",
+    "Set up a virtual environment by going to your project directory and executing the following command. This will create a new virtual environment in a folder named `.venv`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59fa41c8-3535-4d9b-9c30-59e3416ff875",
+   "metadata": {},
+   "source": [
+    "**MacOS/UNIX**\n",
+    "- `python3 -m venv .venv`\n",
```

Comment:
There was no reason, it was just what I do so that my folder looks clean and if I have tested the packages then I would install it in my system. I will edit that part.

Reply:
It's fine - I don't feel strongly on this one.